.gitignore

```diff
@@ -33,3 +33,4 @@ dist
 .settings
 *.retry
 .vagrant
+vagrant/vagrant_data/*.iso
```

Comment:
Please add vagrant/vagrant_data to the Vagrantfile rsync excludes... we don't want to copy this file indiscriminately.